flow360/component/simulation/outputs/outputs.py

```diff
@@ -201,6 +201,32 @@ def check_unique_probe_type(cls, value):
         return _check_unique_probe_type(value, "SurfaceProbeOutput")
 
 
+class TimeAverageProbeOutput(ProbeOutput):
+    """Time average probe monitor output settings."""
+
+    # pylint: disable=abstract-method
```

Comment:
What abstract-method?

Reply:
`Method 'load_point_location_from_file' is abstract in class 'ProbeOutput' but is not overridden in child class 'TimeAverageProbeOutput' (abstract-method)`

Reply:
Hmmm. We still want these model to have this functions though. Could you just add implementation which is just throwing not implemented error? This reminds us that we need to add these in the future. Otherwise nobody will remember this.

Reply:
Shouldn't the implementation be in the base class? It is not meant to be an abstract method, we have this pylint error only because it isn't implemented yet.

Reply:
You are right. Can you add the fake implementation in the base class?